src/components/Offline/Offline.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 
 const Offline = () => (
   <div className="dp-app-container">
+    <Helmet>
+      <meta name="robots" content="noindex,nofollow" />
+    </Helmet>
```

Comment:
It should be also added in `NewFeatures.js`, right?

Reply:
you right, good catch!

Reply:
changed